01.fizzbuzz/fizz_buzz.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+for(i = 1; i < 21; i++){
+  if(i % 5 == 0 && i % 3 == 0){
```

Comment:
後でESLintを導入して機械的に指摘してもらえばいいだけのことではありますが、基本的にこういう感じのスペースの付け方をしたほうがいいと思います。（ESLint導入後に従って貰えばいいので、修正はしなくても良いです）
```suggestion
  if (i % 5 == 0 && i % 3 == 0) {
```